chat-plugins/mafia.js

```diff
@@ -996,13 +1006,13 @@ exports.commands = {
 		reveal: function (target, room, user) {
 			let targetRoom = room;
 			target = target.split(',');
-			if (Rooms(target[0]) && Rooms(target[0]).users[user.userid]) targetRoom = Rooms(target[0]);
+			if (Rooms(target[0]) && Rooms(target[0]).users[user.userid]) targetRoom = Rooms(target.shift());
 			if (!targetRoom || !targetRoom.game || targetRoom.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
 			if (!user.can('mute', null, targetRoom) && targetRoom.game.hostid !== user.userid) return user.sendTo(targetRoom, `|error|/mafia reveal - Access denied.`);
-			let action = targetRoom.id === room.id ? toId(target.join('')) : toId(target[1]);
+			const action = toId(target.join(''));
 			if (!['on', 'off'].includes(action)) return this.parse('/help mafia reveal');
-			if ((action === 'off' && !targetRoom.game.noReveal) || (action === 'on' && targetRoom.game.noReveal)) return user.sendTo(targetRoom, `|error|Revealing of roles is already ${targetRoom.game.noReveal ? 'enabled' : 'disabled'}.`);
-			targetRoom.game.noReveal = action === 'on';
+			if ((action === 'off' && targetRoom.game.noReveal) || (action === 'on' && !targetRoom.game.noReveal)) return user.sendTo(targetRoom, `|error|Revealing of roles is already ${targetRoom.game.noReveal ? 'disabled' : 'enabled'}.`);
+			targetRoom.game.noReveal = action === 'off';
```

Comment:
This dosen't need to be changed...
I already fixed it after someone flipped it last time.

Reply:
I did this so that the command has the same behavior as the bot, ``reveal on`` should enable revealing roles and ``reveal off`` should disable revealing. If you want to have it the other way then sure, but I dont think the command should be called ``reveal`` in that case.

Reply:
https://github.com/Zarel/Pokemon-Showdown/pull/4507/files#diff-b5323b00c0067cd08016aebb74278873R847

i don't see anything wrong with the current help, if you tell it to not "reveal roles on death" they wont and vice versa